app/internal/logger_customizer.py

```diff
@@ -0,0 +1,110 @@
+import json
```

Comment:
Prefer using the project's `config` to configure our logging module

Reply:
Alright, what about catching uvicorn logs? Do we want that? I suppose uvicorn logs its own logs fairly well so I've decided not to add them for now.